README.md

```diff
@@ -79,21 +77,15 @@ skyhookDevApi.register();
 ### Agent Code
 
 ```
-#require "skyhook.agent.nut:1.0.0";
-
-// Device name
-const skyhookDevName = "skyhook-imp-1";
+#require "skyhook.agent.nut:1.0.1";
 
 // Skyhook Precision Location auth key
 const skyhookKey = "Skyhook-Key-From-Skyhook-Dot-Com";
 
-// Skyhook Precision Location URL
-const skyhookUrl = "Location-Url-From-Skyhook>";
-
 // Create Skyhook Precision Location instance
-skyhookAgentApi <- skyhook(skyhookDevName, skyhookKey, skyhookUrl);
+skyhookAgentApi <- skyhook(skyhookKey);
 
-function getMyDeviceLocation(err, location) {
+function device_location(err, location) {
```

Comment:
Oh I see. According to the coding style https://electricimp.com/docs/squirrel/codestyle/#function-and-variable-names lowerCamelCase is highly recommended